packages/account/src/Configs/get-status-badge-config.tsx

```diff
@@ -7,8 +7,8 @@ import { Link } from 'react-router-dom';
 
 const getStatusBadgeConfig = (
     mt5_account_status: TMT5AccountStatus,
-    openFailedVerificationModal?: (selected_account_type: string) => void,
-    selected_account_type?: string,
+    openFailedVerificationModal?: (selected_account_type: any) => void,
+    selected_account_type?: any,
```

Comment:
hi @likhith-deriv why did we keep this as 'any' type

Reply:
a proper mt5_login_list type isn't available